### DISCUSSION TITLE
Examples of aero-structural sensitivity analysis using ADflow and TACS

### DISCUSSION THREAD

**Discussion by lzq1439** [mdolab/adflow#273] [Q&A]
Hello！Are there any examples of aero-structural sensitivity analysis using ADflow and TACS in an aero-stuctural optimization？

**Comment by gawng** [comment]
If you mean code examples of this, yes we do have examples but they are not currently public. If you need to call the sensitivity analysis in ADflow or other solvers in MACH, you need to use `.evalFunctionsSens()` .

**Reply by lzq1439** [reply]
Thank you for your reply! I want to solve the adjoint in an aerostruct problem.  Is there any API that can do this or maybe I should do that by myself?

**Reply by hntter** [reply]
Hi, I am studying to use adflow and tacs to do optimization problem. Can you leave an email so we could discuss any problems?
